manifests/init.pp

```diff
@@ -91,6 +91,7 @@
   $duply_archive_version     = $duplicity::params::duply_archive_version,
   $duply_archive_md5sum      = $duplicity::params::duply_archive_md5sum,
   $duply_archive_url         = undef,
+  $duply_archive_proxy       = undef,       
```

Comment:
Can you please add a quick description of the parameter in the documentation above? It would be interesting to know the expected format as well, e.g. `http://proxy:1234` or just `proxy:1234` ...